src/typeahead/typeahead.ts

```diff
@@ -267,17 +284,13 @@ export class NgbTypeahead implements ControlValueAccessor,
           }
           this._closePopup();
           break;
-        case Key.Escape:
```

Comment:
I prefer `case Key.Escape` to adding this to the service

Reply:
This remains part of the debate: specific vs generic (or local vs service).